db/listener_test.cc

```diff
@@ -211,6 +211,10 @@ class TestFlushListener : public EventListener {
       : slowdown_count(0), stop_count(0), db_closed(), env_(env), test_(test) {
     db_closed = false;
   }
+
+  virtual ~TestFlushListener() {
+    prev_fc_info_.status.PermitUncheckedError();  // Ignore the status
```

Comment:
I am just wondering if this should not be in the constructor (inline with some of your recent suggestions)?